src/revlanguage/main.cpp

```diff
@@ -28,14 +28,38 @@ using po::variables_map;
 
 std::string usage()
 {
-    return "Usage: rb [OPTIONS]\n       rb [OPTIONS] <file1> [<file2> ...]\n";
-    // Other usages not mentioned
+    std::string usage =
+        "Usage: rb [OPTIONS]\n"
+        "       rb [OPTIONS] <file1> [<file2> ...] [--args <arg1> [<arg2> ...]]";
+
+    return usage;
+}
+
+
+std::string usage_examples()
+{
+    std::string usage_examples =
+        "Usage examples:\n"
+        "   1. rb\n"
+        "   2. rb --args 1 2\n"
+        "   3. rb script.Rev\n"
+        "   4. rb --file script.Rev                         # equivalent to 3\n"
+        "   5. rb --cmd script.Rev                          # equivalent to 3 and 4 but runs in batch mode\n"
+        "   6. rb script.Rev --args 1 2\n"
+        "   7. rb --args 1 2 --file script.Rev              # equivalent to 6\n"
+        "   8. rb --cmd script.Rev 1 2                      # equivalent to 6 and 7 but runs in batch mode\n"
```

Comment:
I think the usage examples would be clearer if all the uses of `--cmd` were together, at the end.
- rb --cmd script.Rev
- rb --cmd script.Rev 1 2
- rb --cmd script.Rev 1 2 --file script2.Rev

Reply:
Done